app/slackimport.go

```diff
@@ -659,7 +662,9 @@ func (a *App) SlackImport(fileData multipart.File, fileSize int64, teamID string
 				} else {
 					posts[channel] = append(posts[channel], newposts...)
 				}
-			} else if len(spl) == 3 && spl[0] == "__uploads" {
+				continue
+			}
+			if len(spl) == 3 && spl[0] == "__uploads" {
```

Comment:
I'd suggest not making this change: we aren't doing it on line 653 above, and this isn't really about error handling per se.

Reply:
I was not sure about this change; it seemed to make the code more readable, as there is another ```if``` enclosed just before. But I agree that it is a question of preference and that this kind of change does not belong in this commit.